single_tech_samples/synapseanalytics/sample3_integration_testing/tests/integrationtests/README.md

```diff
@@ -0,0 +1,71 @@
+# Integration test suite for ADF
```

Comment:
let's add a TOC via the appropriate vscode extension

Reply:
done